From 5dad6a5fb5a00d128dc958135336ac14c1490ece Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Thu, 16 May 2024 18:01:30 +0200
Subject: [PATCH] arm: dts: cn9132-cex7: enable correct usb ports routed to b2b
 connectors

CN9132 CEX7 routes the following usb signals:
A45/A46: CP0-USB0
B42/B43: CP0-USB1
A42/A43: CP1-USB0
B45/B46: CP2-USB0
C6/C7/D6/D7: CP2-USBSS0

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm/dts/cn9131-cex7.dtsi | 8 ++++++++
 arch/arm/dts/cn9132-cex7.dtsi | 4 ----
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/cn9131-cex7.dtsi b/arch/arm/dts/cn9131-cex7.dtsi
index 6217b1e6dd..88e860b105 100644
--- a/arch/arm/dts/cn9131-cex7.dtsi
+++ b/arch/arm/dts/cn9131-cex7.dtsi
@@ -163,3 +163,11 @@
         num-lanes = <2>;
         status = "okay";
 };
+
+&cp1_utmi0 {
+	status = "okay";
+};
+
+&cp1_usb3_0 {
+	status = "okay";
+};
diff --git a/arch/arm/dts/cn9132-cex7.dtsi b/arch/arm/dts/cn9132-cex7.dtsi
index 514ddfa7d4..0ff5a39622 100644
--- a/arch/arm/dts/cn9132-cex7.dtsi
+++ b/arch/arm/dts/cn9132-cex7.dtsi
@@ -55,7 +55,3 @@
 &cp2_utmi0 {
 	status = "okay";
 };
-
-&cp2_utmi1 {
-	status = "okay";
-};
-- 
2.35.3

